lib/nunjucks/govuk-prototype-kit/internal/views/manage-prototype/plugins.njk

```diff
@@ -32,42 +18,46 @@
         <ul class="govuk-list govuk-prototype-kit-manage-prototype-plugin-list-plugin-list">
           {% for plugin in group.plugins %}
             <li class="govuk-prototype-kit-manage-prototype-plugin-list-plugin-list__item">
-              <div class="govuk-prototype-kit-manage-prototype-plugin-list-plugin-list__item-name">
+              <span class="govuk-prototype-kit-manage-prototype-plugin-list-plugin-list__item-name">
                 {{ plugin.name }}
                 {% if plugin.scope %}
-                <br/>
+                  <br/>
                   <span class="govuk-caption-m">From {{ plugin.scope }}</span>
                 {% endif %}
                 {% if plugin.upgradeLink %}
                   <br/>
                   v{{ plugin.latestVersion }} available
                 {% endif %}
-              </div>
-              <div class="govuk-prototype-kit-manage-prototype-plugin-list-plugin-list__item-version">
+              </span>
+              <span class="govuk-prototype-kit-manage-prototype-plugin-list-plugin-list__item-version">
                 {{ plugin.installedVersion or plugin.latestVersion }}
-              </div>
-              <div class="govuk-prototype-kit-manage-prototype-plugin-list-plugin-list__item-links">
+              </span>
+              <span class="govuk-prototype-kit-manage-prototype-plugin-list-plugin-list__item-buttons">
                 {% if not plugin.installedVersion %}
-                  <a class="govuk-prototype-kit-manage-prototype-plugin-list-plugin-list__item-link"
-                     href="{{ plugin.installLink }}">Install<span class="govuk-visually-hidden"> {{ plugin.name }}</span></a>
+                  <button id="install-{{ plugin.packageName }}" class="govuk-button govuk-button--secondary"
+                     data-module="govuk-button"
+                     formaction="{{ plugin.installLink }}">Install<span class="govuk-visually-hidden"> {{ plugin.name }}</span></button>
```

Comment:
Can we use the GOV.UK Frontend Nunjucks macros here?

```suggestion
                  {{ govukButton({
                    classes: "govuk-button--secondary",
                    attributes: { id: "install-" + plugin.packageName, formaction: plugin.installLink }
                  }) }}
                  <button id="install-{{ plugin.packageName }}" class="govuk-button govuk-button--secondary"
                     data-module="govuk-button"
                     formaction="{{ plugin.installLink }}">Install<span class="govuk-visually-hidden"> {{ plugin.name }}</span></button>
```

Reply:
Done